plugins/modules/public_cloud_instance_interface.py

```diff
@@ -76,21 +78,21 @@ def run_module():
     instance_id = module.params['instance_id']
     state = module.params['state']
     interface_ip = module.params['interface_ip']
-    interface_network_id = module.params['interface_network_id']
+    interface_openstack_id = module.params['interface_openstack_id']
 
     if module.check_mode:
         module.exit_json(msg="Ensure interface {} on {} is {} on instance id {} - (dry run mode)"
-                         .format(interface_ip, interface_network_id, state, instance_id),
+                         .format(interface_ip, interface_openstack_id, state, instance_id),
                          changed=True)
 
     if state == 'absent':
         # Need to get the interface id (via /cloud/project/{serviceName}/instance/{instanceId}/interface).
         # How to manage multiple interfaces ?
         module.fail_json(msg="Removing an interface is not yet implemented")
     if state == 'present':
-        result = client.wrap_call("POST", f"/cloud/project/{service_name}/instance/{isinstance}/interface",
```

Comment:
`isinstance` is an error in the last PR. It should have been `instance_id` instead